sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCast.scala

```diff
@@ -676,7 +676,7 @@ object GpuCast extends Arm {
 
       /* -------------------------------- helper functions -----------------------*/
 
-      /**
+      /*
        * cast all not-null elements in a child column to string type <p>
        * add `' '` to all elements when `legacyCastToString = true`
        * @param child child column of an array column
```

Comment:
Nit: Odd to be using Scaladoc directives in a comment that is not marked for Scaladoc.  Someone may come along later and assume it was intended to be a Scaladoc comment and "fix" it to be so.

This comment applies to the other instances.

Reply:
Thank you for your advice, I will fix it